### ISSUE TITLE
Editor

### ISSUE THREAD

**Issue by ksolo** [CodePlatoon/curriculum#1]
From the Curriculum File:
Become familiar with editor (What editor do we want to use? Probably Atom or Emacs, might depend on dev env, eg are we doing cloud based
\--------------------------------------------------

I would recommend Atom or Sublime Text. Even of these two I would go Atom. Atom is open source so students could even take a look at how something works. It's also become my default for editing Markdown files with the live preview.

**Comment by JoshCheek** [comment]
Yeah, I definitely prefer Atom over Sublime, and the SiB integration is best in Atom over any of the other editors. Only reason I haven't switched to it from Textmate is that TM has some really nice keybindings.

My only real apprehension is that it allows a student to use the mouse, and I would rather have them struggle in the beginning than develop a habit like this, which will dramatically impede them in the long run, if they don't address it. I worry about things like this a lot, hence making them [practice typing](https://github.com/CodePlatoon/curriculum/blob/master/prework-practice-typing.md) ^_^

**Comment by ksolo** [comment]
Ultimately I'm not 100% opposed to using the mouse. That being said, I think you need to invest the time to learn your editor/IDE well and use as many shortcuts as possible.
